protocol/proto/metadata.proto

```diff
@@ -300,7 +300,12 @@ message AudioFile {
         MP3_160_ENC = 7;
         AAC_24 = 8;
         AAC_48 = 9;
+        AAC_160 = 10;
+        AAC_320 = 11;
+        MP4_128 = 12;
+        OTHER5 = 13;
```

Comment:
@roderickvd You mentioned in the ticket OTHERS5 = 13 can be deleted since it's "FLAC_FLAC = 16 now.. Is my understanding correct?

Reply:
I was on mobile then and am now again, so sorry if I'm causing confusion. I remember that somewhere it said 0xa or 0xd and I want to be sure that we're not doubling anything in decimal format (10, 13). I have a vague memory that they repurposed some enum variant to FLAC which did not exist in older protobufs.

Reply:
No problem.... While merging v0.4.2 into current 0.5.0-dev I have seen that there is conses up to MP3_160_ENC = 7 which is same for both version. 

But I see a conflict for the following ones:
AAC_24 = 8;             //this is mapped to MP4_128_DUAL in v0.4.2, but also AAC_24 (commented out)
AAC_48 = 9;             //this is mapped to OTHER3 in v0.4.2, but also AAC_48 (commented out)

Since the following numeric values (10, 11, 12) refers to AAC and the fact that AAC_24 and AAC_48 are already existent as comment in v0.4.2, I mapped them to AAC_24 = 8 and AAC_48 = 9. Don't know if this is correct.

These are the ones which are missing in v0.5.0-dev:
AAC_160 = 10;         //ok, missing in v0.5.0
AAC_320 = 11;         //ok, missing in v0.5.0
MP4_128 = 12;         //ok, missing in v0.5.0
OTHER5 = 13;          //ok, missing in v0.5.0

That one is missing in v0.4.2:
FLAC_FLAC = 16;    //ok, missing in v0.5.0 

From a coding perspective the missing ones is just a pure merge. I cannot judge if they are correct anyway with respect to Spotify's spec. For the conflicting AACs I just used the variant which seems to be most reasonable IMHO. 

Maybe there is a history behind and the mapping has also changed from Spotify's side (refers "v4" comment in v0.4.2 to kind of a protocol versioning?). I have no clue about that. Would be great if someone with deeper knowledge can review the values used in my pull request and give an OK or any better mapping. Even though my understanding is, that beside OGG nothing else is really used at the moment.

Reply:
Makes sense to me. Indeed some of the mappings changed over time on Spotify's end.